src/test/java/com/rakovets/course/java/core/practice/oop/principles/battle_ground/ArcherTest.java

```diff
@@ -0,0 +1,121 @@
+package com.rakovets.course.java.core.practice.oop.principles.battle_ground;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class ArcherTest {
+    @Test
+    void Archer() {
+        Archer legolas = new Archer("Legolas", 750);
+
+        Assertions.assertEquals(750, legolas.getHealth());
+        Assertions.assertEquals("Legolas", legolas.getName());
+    }
+    @Test
+    void ArcherName() {
```

Comment:
"Violation: 'camelCase' was not used for method naming
Solution: Rename method
"